superset-frontend/cypress-base/cypress/integration/explore/visualizations/big_number.js

```diff
@@ -56,8 +56,8 @@ export default () =>
 
     it('should work', () => {
       verify(BIG_NUMBER_FORM_DATA);
-      cy.get('.chart-container .header_line');
-      cy.get('.chart-container .subheader_line');
+      cy.get('.chart-container .header-line');
+      cy.get('.chart-container .subheader-line');
```

Comment:
`explore/visualizations/index.test.js` was probably unintentioally disabled by https://github.com/apache/incubator-superset/pull/6241 .

It has since stopped running for over a year now. A lot of these tests are now broken. It's easy to add them back, but we may want to do the fix in another PR. Keep the changes for BigNumber here just for reference.